### ISSUE TITLE
Smoothing mutation data with PPI networks

### ISSUE THREAD

**Issue by messersc** [BIMSBbioinfo/maui#3]
1. I can't find the parameter alpha that was used to run netsmooth to generate the smoothed mutation matrix. It would be nice to include it in the manuscript.

2. The choice of protein-protein network to use is essentially another hyperparameter. Would another PPI network perform similar? And even more interesting, how would a random network of similar density perform? Looking at the methods part and the code, I am be a bit concerned that batch normalizing the binary mutation input features (batch size n=50 per default) could be problematic. Am I missing something?

**Comment by jonathanronen** [comment]
Thanks for these comments too, @messersc. For 1., I've now stated in the manuscript that I use alpha 0.7, which I and others have found to be a good rule of thumb for mutation data. For other network sources (and fake network sources), I suggest you have a look at the netsmooth paper [1] where we discussed this.

As for batch normalizing mutations, unless you specify your concerns about what might be problematic, I'm not sure what I am missing :)

[1]: https://f1000research.com/articles/7-8/v3
